lib/tasks/bulk_upload.rake

```diff
@@ -0,0 +1,28 @@
+require 'csv'
+namespace :bulk_upload do
+  desc 'Bulk upload facilites from CSV; Example: rake "bulk_upload:facilities_from_csv[path/to/file]"'
+  task :facilities_from_csv, [:facilities_file] => :environment do |_t, args|
+    facilities_file = args.facilities_file
+    created_facilities = 0
+    CSV.foreach(facilities_file, headers: true) do |row|
```

Comment:
Ideally this uses `open-uri` so we can pass a URL, not just a file we have to upload somehow.

You can see my approach in my very-similar-to-this PR:
https://github.com/simpledotorg/simple-server/pull/101

Reply:
:+1: